app/views/reservations/users_reservations.html.erb

```diff
@@ -9,7 +9,8 @@
     <tbody>
       <tr>
         <td><%= image_tag gravatar_url(@user.email, 64), alt: "" %> </td>
-        <td> <% if @user.name? %><%= @user.name %><br><% end %> <%= @user.email %></td>
+        <td> <% if @user.name? %><%= @user.name %><br><% end %> <%= @user.email %><br> <%=time_ago_in_words( @user.birghtday) %><br><%= (Time.now.year - current_user.birghtday.year) %>
```

Comment:
Zauważyłem, że powtarzasz w wielu miejscach w kodzie coś takiego:
```
(Time.now.year - current_user.birghtday.year)
```

W przypadku widoku to raczej trzeba przenieść do helpera, bo `erb` staje się przez to mało nieczytelny.
W przypadku UsersController to trzeba dać do serwisu. Ewentualnie stworzyć po prostu metodę w klasie User. Powinna ona mieć taką treść:

```
def age
  ((Date.today - birghtday) / 365).to_i
end
```
Tak skonstruowana metoda da Ci dokładny wiek, odejmowanie roku urodzenia jest mało dokładne i nie pokryje Ci wielu przypadków.

i kolejna metoda:
```
def adult?
  age > 18
end
```

Wtedy znikną Ci te duplikacje kodu. W widoku wystarczy, że wywołasz `@user.age`, a w kontrolerze zamiast tworzyć metodę `adult?` możesz użyć bezpośrednio na current_user.